hathor/cli/main.py

```diff
@@ -91,6 +92,7 @@ def __init__(self) -> None:
         self.add_cmd('dev', 'x-export', db_export, 'EXPERIMENTAL: Export database to a simple format.')
         self.add_cmd('dev', 'x-import', db_import, 'EXPERIMENTAL: Import database from exported format.')
         self.add_cmd('dev', 'replay-logs', replay_logs, 'EXPERIMENTAL: re-play json logs as console printted')
+        self.add_cmd('dev', 'run_from_logs', run_from_logs, 'replay the same blocks and transactions in the log')
```

Comment:
What about naming it `load_from_logs` (or `load-from-logs`, we're not consistent with using `-` or `_`)?

I was initially confused by the name since this isn't replaying the *logs* themselves (that's what `replay-logs` does), this is replaying the *loading* of blocks/transactions as they were found in a log dump. And also because the node doesn't actually *run* after the blocks/transactions are added/loaded.